stdlib/2and3/imghdr.pyi

```diff
@@ -1,16 +1,21 @@
-from typing import overload, Union, Text, BinaryIO, Optional, Any, List, Callable
-import sys
 import os
+import sys
+from typing import Any, BinaryIO, Callable, List, Optional, Protocol, Text, Union, overload
+
+class _ReadableBinary(Protocol):
+    def tell(self) -> int: ...
+    def read(self, size: int = ...) -> bytes: ...
+    def seek(self, offset: int) -> Optional[int]: ...
```

Comment:
I didn't add the `whence` parameter here as we see on `IO` since it's not used in `imghdr`. Hopefully this is the right way to implement a protocol - i.e. it doesn't seem to cause any issues when the class that we pass to `imghdr` does implement a `whence` parameter.

Reply:
Exactly.